go/libraries/doltcore/sqle/dfunctions/dolt_merge.go

```diff
@@ -293,7 +319,7 @@ func executeNoFFMerge(
 	dbName string,
 	ws *doltdb.WorkingSet,
 	dbData env.DbData,
-	//headCommit, mergeCommit *doltdb.Commit,
+	// headCommit, mergeCommit *doltdb.Commit,
```

Comment:
(minor) might as well remove this old comment; doesn't seem like it's doing anything helpful.

```suggestion
```